### ISSUE TITLE
"Next" feature/front matter present in old docs site breaks new site

### ISSUE THREAD

**Issue by asachs01** [sensu/sensu-docs-site#246]
**Example page**: https://raw.githubusercontent.com/sensu/sensu-docs/master/docs/1.2/installation/summary.md

**Content that causes issue**: 
```
next:
  url: "getting-started"
  text: "Getting Started Guide"
```

**Description of issue**: Adding in the above front matter to a page breaks the page and results in a `404`. The error message that Hugo spits out is:
```
Building sites … ERROR 2018/03/13 09:50:32 Error while rendering "page" in "sensu-co
re/0.29/installation/": template: _default/single.html:37:11: executing "_default/si
ngle.html" at <partial "footer" .>: error calling partial: template: partials/footer
.html:18:21: executing "partials/footer.html" at <split .Params.next "...>: error ca
lling split: unable to cast map[interface {}]interface {}{"url":"getting-started", "
text":"Getting Started Guide"} of type map[interface {}]interface {} to string
```

**Why does this matter**: In the previous docs site, we have established a flow in the sections that guides Sensu users through the docs, or directs them to specific platforms. With moving to the use of the platform selector, I can see this not being as useful. However, being able to guide users through the process of installing & configuring Sensu is something that I don't think we can lose. @mbbroberg @cwjohnston do you have any thoughts on this ("this" being the ability for the docs site to provide a more guided experience)?

**Comment by cwjohnston** [comment]
Per [this post](https://discourse.gohugo.io/t/question-regarding-next-prev-navigation/1055/1) I think hugo expects `next:` and `prev:` to have string values.

Also looks like there's a way to do this using variables, provided we arrange the documents in a "section": https://discourse.gohugo.io/t/question-regarding-next-prev-navigation/1055/6

**Comment by cwjohnston** [comment]
Closed by #249, in which we figured out how to make next/prev work in this new context 👍
